src/Mobs/Path.cpp

```diff
@@ -0,0 +1,391 @@
+// TODO uniq_ptr instead of ptr whenever possible
+
+#ifndef COMPILING_PATHFIND_DEBUGGER
```

Comment:
MSVC builds will not work unless the very first preprocessor instruction in each CPP file is the `#include "Globals.h"` line. You need to work around that in your PathFinderDebugger project - provide an empty Globals.h, for example.